lms/djangoapps/debug/management/commands/dump_xml_courses.py

```diff
@@ -31,7 +31,6 @@ def handle(self, *args, **options):
             default_class='xmodule.hidden_module.HiddenDescriptor',
             load_error_modules=True,
             xblock_mixins=settings.XBLOCK_MIXINS,
-            xblock_select=settings.XBLOCK_SELECT_FUNCTION,
```

Comment:
Side note: I wasn't able to run this management command, as it doesn't seem to be working without `argparse`. Using `BaseCommand.args` [was deprecated in Django 1.8](https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#django.core.management.BaseCommand.args).

Reply:
Optional: Please feel free to just delete this entire file. The XMLModuleStore was killed off as a functional thing sometime in late 2015. Nothing about this command should work.